src/qibo/backends/clifford.py

```diff
@@ -39,12 +40,9 @@ def __init__(self, engine=None):
             engine = _check_backend(engine)
             engine = engine.name if engine.platform is None else engine.platform
 
-        class CliffordEngine:
-            pass
-
-        self.engine = CliffordEngine()
-        for method in dir(_clifford_operations):
-            setattr(self.engine, method, getattr(_clifford_operations, method))
+        spec = find_spec("qibo.backends._clifford_operations")
```

Comment:
Why did you decide for the existing module?

Reply:
I don't know it looked simpler to me and  I don't think there is any difference with the empty module.

Reply:
The differences are that:
- it is non-empty, that is an advantage
- it could be registered in `sys.modules`, or somehow hooked to the `import` mechanism
    - if that's the case, you would be permanently modifying that module, loosing access to the original content (same of a static import)

Reply:
Anyhow, I see it as a temporary solution. If point 2. does not apply (you could modify that module, without modifying the module imported statically), then it's alright.